### PR TITLE
feat: sets ft for local outputpanel buffer

### PR DESCRIPTION
**Pull request by megalithic**
I like to set up custom things based on `filetype`, such as binding `q` to quit in normal mode, for a specific filetype, or list of filetypes.

This opens up that possibility.

### CODE REVIEW

**Review by mhanberg — COMMENTED** [review]
let's go with all lowercase. 

Have you tested this locally?

File: lua/output_panel/init.lua [hunk 1]
```diff
@@ -40,6 +40,7 @@ function M.panel()
 
   vim.api.nvim_win_set_buf(0, output_panel_bufnr)
   vim.api.nvim_win_set_height(0, 30)
+  vim.opt_local.filetype = "OutputPanel"
```
Comment (mhanberg) [inline lua/output_panel/init.lua]: Also can you change it to vim.bo.filetype? If that is the one that works for filetype.

**Comment by megalithic** [comment]
> let's go with all lowercase.
> 
> Have you tested this locally?

I have indeed! [See my config, to see how/where I use this](https://github.com/megalithic/dotfiles/commit/7ae4be89874535cbe656a01ba5e0a47cc7ff87a7).

Thanks for the great plugin!

**Comment by mhanberg** [comment]
Thank you for the contribution!

**Pull request merged**